tests/run_notebook.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbformat import read
+
+
+def execute_notebook(notebook_path) -> None:
+    notebook = read(notebook_path, as_version=4)
+    execute_preprocessor = ExecutePreprocessor(timeout=600)
```

Comment:
minor thing, please set a higher timeout (1800), since some cells could be slow to execute in some large notebooks.